Update website content

rileyhgrant opened the pull request:
Updates website content
- Adds 'human' to about page
- Removes member of SAB who is no longer member
- Adds 'tm' to gnomAD where relevant
- Moves and edits Open Science Policy
- Adds gnomAD videos FAQ
- Adds "Federated gnomAD" page
- Rework citations page (re-organize, add new papers, use component for reusable citation)
- Updates the Team page - moves Steve to Alumni :(

phildarnowsky-broad commented on browser/about/about.md:
```diff
@@ -1,4 +1,4 @@
-The Genome Aggregation Database (gnomAD), originally launched in 2014 as the Exome Aggregation Consortium (ExAC), is the result of a coalition of investigators willing to share aggregate human exome and genome sequencing data from a variety of large-scale sequencing projects, and make summary data available for the wider scientific community. The project is overseen by co-directors Heidi Rehm and ...
+The Genome Aggregation Database (gnomAD™), originally launched in 2014 as the Exome Aggregation Consortium (ExAC), is the result of a coalition of investigators willing to share aggregate human exome and genome sequencing data from a variety of large-scale sequencing projects, and make summary data available for the wider scientific community. The project is overseen by co-directors Heidi Rehm an ...
```
Asking for my own information here, is there documentation about when it's necessary to use the :tm:?

rileyhgrant replied:
I think the main intent was to have it in a prominent location on policies, I can share the google doc from Heidi/Sam that I've been working off of -- I've pretty much just been following their edits for these content things.

rileyhgrant replied:
https://docs.google.com/document/d/1Stj4N9xXu5KvYeGgCuJxvvUBOjhP4eqjErryY7ZXNdc

phildarnowsky-broad commented on browser/about/policies/terms.md:
```diff
@@ -1,15 +1,21 @@
 This data set has been subjected to extensive quality control, but variant calling and filtering from short-read sequencing data is an imperfect and probabilistic process, so many errors no doubt remain. If you spot any results that seem impossible, or suggest some kind of serious processing or variant-calling artifact, don't panic: use the "report variant" form on the corresponding variant page  ...
 
-The primary data from the gnomAD™ exomes and genomes are available free of restrictions under the [Creative Commons Zero Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/). This means that you can use it for any purpose without legally having to give attribution. However, we request that you actively acknowledge and give attribution to the gnomAD™ project, and link back ...
+The primary data from the gnomAD exomes and genomes are available free of restrictions under the [Creative Commons Zero Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/). This means that you can use it for any purpose without legally having to give attribution. However, we request that you actively acknowledge and give attribution to the gnomAD project, and link back t ...
```
Is the removal of the :tm: here intentional?

rileyhgrant replied:
Yeah, after double checking with Sam, adding the 'TM' everywhere was just cluttering things, the 'TM' still shows up on the header to this section, and that was deemed sufficient to get the message across.

phildarnowsky-broad commented on browser/src/FederationPage.tsx:
```diff
@@ -0,0 +1,236 @@
+  <InfoPage>
+    <DocumentTitle title="Federated gnomAD" />
+    <PageHeading
+      // @ts-expect-error
```
What's this for? We should avoid adding new `ts-expect-error`s if we can get out of it.

phildarnowsky-broad commented on browser/src/FederationPage.tsx:
```diff
@@ -0,0 +1,236 @@
+import styled from 'styled-components'
+import { Link, PageHeading } from '@gnomad/ui'
+
+// @ts-ignore - TS2307 Cannot fine module ... or its corresponding type declarations.
```
I'm guessing that we have to use `ts-ignore` here liberally because these are assets as opposed to code?

rileyhgrant replied:
Yeah exactly, these `@ts-ignore`s are currently annoyingly used in any place like this, e.g. the Stats page or Team page also where are using import statements for assets.

phildarnowsky-broad commented on browser/src/FederationPage.tsx:
```diff
@@ -0,0 +1,236 @@
+                <td>{dataset.samples}</td>
+                <td>
+                  {dataset.link ? (
+                    // @ts-expect-error
```
...whereas I'd guess this particular `ts-expect-error` is due to the known issue with elements of GBTK not having `children` declared?

rileyhgrant replied:
Yep :(

phildarnowsky-broad commented on browser/src/FederationPage.tsx:
```diff
@@ -0,0 +1,236 @@
+
+    <CenteredContainer>
+      <LogoContainer>
+        <SingleLogo alt="" src={CanadaGnomadLogo} />
```
Could we set these alts to "[Name of federated project] logo" or similar?

rileyhgrant replied:
Ah, good call, I should really leave myself `TODO` comments to clean up before submitting for a review, instead of assuming I'll remember to do the things. Will fix.

phildarnowsky-broad reviewed (commented):
There's a couple of things I'm dubious about, I'm guessing you had good reasons for most or all of them but I'd like to understand the reasoning too.

rileyhgrant commented:
Heya @phildarnowsky-broad 

Added two fixup commits
- Removed the one unneccesary `@ts-expect-error` that was called out
- Added alt text for the logos -- thanks for the catch!

rileyhgrant commented:
Just noting here that I'm waiting until Friday to merge this -- at that time at least one of the groups on the Federated page will have voted on a different name, so that text will have to change.
